src/js/constants/AppStatus.js

```diff
@@ -9,4 +9,4 @@ var AppStatus = {
   WAITING: 4
 };
 
-module.exports = AppStatus;
+module.exports = Object.freeze(AppStatus);
```

Comment:
Sorry – I just realised I did not understand what is going on in #119 . Nevermind this comment.